src/Evolution/Executables/GeneralizedHarmonic/EvolveGeneralizedHarmonic.hpp

```diff
@@ -326,41 +335,60 @@ struct EvolutionMetavars {
       Initialization::Actions::AddComputeTags<
           tmpl::list<evolution::Tags::AnalyticCompute<
               volume_dim, analytic_solution_tag, analytic_solution_fields>>>,
-      GeneralizedHarmonic::gauges::Actions::InitializeDampedHarmonic<
-          volume_dim>,
-      GeneralizedHarmonic::Actions::InitializeConstraints<volume_dim>,
       dg::Actions::InitializeMortars<boundary_scheme, true>,
       Initialization::Actions::DiscontinuousGalerkin<EvolutionMetavars>,
       Initialization::Actions::RemoveOptionsAndTerminatePhase>;
 
-  using component_list = tmpl::list<
+  using initialize_initial_data_dependent_quantities_actions = tmpl::list<
+      GeneralizedHarmonic::gauges::Actions::InitializeDampedHarmonic<
+          volume_dim>,
+      GeneralizedHarmonic::Actions::InitializeConstraints<volume_dim>,
+      Parallel::Actions::TerminatePhase>;
+
+  using component_list = tmpl::flatten<tmpl::list<
```

Comment:
do we need the `flatten` still here?

Reply:
The `component_list` is not currently flattened in the backend, it's being retrieved directly from the `Metavariables` in multiple places